grails-app/views/stockMovement/_summary.gsp

```diff
@@ -1,3 +1,4 @@
+<%@ page import="org.pih.warehouse.DateFormatName" %>
```

Comment:
This should be in a package like "core". Unless there's a better place for it.

Reply:
Or "util" if people feel more strongly about that.

Reply:
I didn't know where it should be placed, so thanks for this comment.

Reply:
Yeah let's move it to `src/main/groovy/org/pih/warehouse/core` since that's where most of the enums are located. And let's change the name of the enum to just DateFormat. Or if that's problematic because of java.text.DateFormat, then maybe DateFormatCode. Something is bothering me about DateFormatName. If anyone has any other suggestions please add it here.

Reply:
I'm trying something new with respect to making decisions as a team, so please take this poll when you have a minute.

https://github.com/openboxes/openboxes/discussions/4693

@alannadolny @awalkowiak @EWaterman @drodzewicz @kchelstowski